checkmarx-ast-eclipse-plugin/src/com/checkmarx/eclipse/views/DataProvider.java

```diff
@@ -366,33 +366,95 @@ private List<DisplayModel> addResults(String scanId, Map<String, List<DisplayMod
 		return returnList;
 	}
 
+
+	/**
+	 * Creates a clean Result object with decoded HTML entities
+	 *
+	 * @param resultItem Original result object
+	 * @return New Result object with cleaned values
+	 */
+	private Result createCleanResult(Result resultItem) {
```

Comment:
do we have a test to this new logic?

Reply:
Unfortunately we don't have a testing infrastructure there, only a UI one. We need to open a separate task for that.